libs/kotaemon/pyproject.toml

```diff
@@ -37,6 +37,7 @@ dependencies = [
     "llama-index-vector-stores-chroma>=0.1.9",
     "llama-index-vector-stores-lancedb",
     "llama-index-vector-stores-milvus",
+    "llama-index-vector-stores-qdrant",
```

Comment:
Let's move this to [adv] since currently we have quite a few variance of vectorstores, which can increase installation time and storage for basic users, while they are not necessary use it.

Reply:
[`5420a54` (#260)](https://github.com/Cinnamon/kotaemon/pull/260/commits/5420a54360b41443abaca22e5eda5cb861dac5c1)